mqtt/mqtt-bridge/src/client.rs

```diff
@@ -21,9 +21,10 @@ use crate::{
 };
 
 const DEFAULT_TOKEN_DURATION_MINS: i64 = 60;
-const DEFAULT_MAX_RECONNECT: Duration = Duration::from_secs(5);
+const DEFAULT_MAX_RECONNECT: Duration = Duration::from_secs(60);
```

Comment:
Increased default to 1min because it was retrying too often to reconnect

Reply:
Can we add exponential backoff with randomized intervals?

Reply:
this is the max backoff, before it reaches it is using exponential but it starts with 1 sec not random value. client doesn't expose a setting for starting interval, do you thing should be added?